app/models/appointment.rb

```diff
@@ -1,7 +1,10 @@
 class Appointment < ApplicationRecord
   include PgSearch
 
-  pg_search_scope :search, against: %i(id first_name last_name)
+  pg_search_scope :search, against: %i(id first_name last_name), associated_against: {
+    guider: [:name]
```

Comment:
Actually they brought up the search thing yesterday, ideally we would have partial matching on this stuff. But I thought I'd bring that up later in case it's a big change 🙈